planet/cli/subscriptions.py

```diff
@@ -78,14 +78,22 @@ def subscriptions(ctx, base_url):
     multiple=True,
     default=None,
     help="Select subscriptions in one or more states. Default is all.")
+@click.option(
+    '--source-type',
+    default=None,
+    help=
+    "Filter subscriptions by source type. See documentation for all available types. Default is all."
```

Comment:
yapf --in-place filename (from [CONTRIBUTING.md](https://github.com/planetlabs/planet-client-python/blob/main/CONTRIBUTING.md#yapf)) is evidently insufficient, as it produced:
```
    help=
    "Filter subscriptions by source type. See documentation for all available types. Default is all."
```
Which [failed linting](https://github.com/planetlabs/planet-client-python/actions/runs/11584616504/job/32252092775?pr=1074)

Reply:
I was able to fix manually, but it'd be nice if our automated linting actually worked as described.